assets/css/index.css

```diff
@@ -44,6 +44,7 @@ footer {
 }
 
 #particles-js {
+  background-image: url('../img/stardust-bg.png');
```

Comment:
unclear where this image came from, but probably also needs to be added as an asset